lib/vm.js

```diff
@@ -18,19 +18,21 @@ const realRunInContext = Script.prototype.runInContext;
 
 Script.prototype.runInThisContext = function(options) {
   if (options && options.breakOnSigint) {
-    return sigintHandlersWrap(() => {
+    const sigintHandler = () => {
```

Comment:
I am a bit :confused: about the name here… this function does not handle any kind of signal; it’s the exact opposite, the function is run undisturbed by SIGINTs. (The comment/code for `sigintHandlersWrap` has a bit more information.) So maybe you could call it `realRunInThisContext` or `runScriptUninterruped` or something like that?